### DISCUSSION TITLE
GOTH Stack!

### DISCUSSION THREAD

**Discussion by atridadl** [dobicinaitis/tailwind-cli-extra#1] [Show and tell]
First: Thank you so much for doing this! I was going to try this myself yesterday in an attempt to get all JS out of my Go web app project, but found that you had the same idea just a day earlier! This is such a time saver!

I include the binaries now in my Go + HTMX + Tmpl project template: https://github.com/atridadl/goth.stack

**Comment by dobicinaitis** [comment]
Thanks, I'm happy to hear that it's useful to you! :blush:
